helpers/docker-compose.yaml

```diff
@@ -0,0 +1,50 @@
+version: '3.9'
+
+services:
+  postgres:
+    image: postgres:12
+    ports:
+      - "5432:5432"
+    environment:
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=postgres
+      - POSTGRES_DB=notifications
+      - PGUSER=postgres
+    healthcheck:
```

Comment:
You could just leave the healthcheck out ?

Reply:
That doesn't work. I am getting "ERROR: for flyway2  Service "flyway" is missing a healthcheck configuration" 
when I remove it. Have to put one healtcheck on a service if I want to depends_on it.

The problem is, that I need "flyway" to be executed before "flyway2" because the first one executes the initial flyway stuff we also execute for every other build. And flyway2 should run after that and insert test data.